README.rst

```diff
@@ -38,7 +38,17 @@ The page is currently hosted on http://localhost:1990/
 Environment Variables/Setup Notes
 ---------------------------------
 
-This MFE will use a REST API from the Credentials IDA located at credentials/apps/records/rest_api
+Currently, this MFE is not intergrated into the devstack, and must be run locally. This MFE requires credentials to be running, and will use a REST API from the Credentials IDA located at `credentials/apps/records/rest_api`.
+
+Credentials uses 2 enviroment variables to link to this MFE:
+- USE_LEARNER_RECORD_MFE -- Toggles the navigation in credentials to redirect to this MFE
+- LEARNER_RECORD_MFE_RECORDS_PAGE_URL -- The URL for the base URL of this MFE
+More details for these flags can be found in the base configuration of credentials :`credentials/settings/base`
+
+This MFE has 2 flags of its own:
+- SUPPORT_URL_LEARNER_RECORDS -- A link to a help/support center for learners who run into problems whilst trying to share their records
+- USE_LR_MFE -- A toggle that when on, uses the MFE to host shared records instead of the the old UI inside of credentials
```

Comment:
I think the `-` may need a space in front of them to make the bullet?

![Screen Shot 2022-10-12 at 1 36 27 PM](https://user-images.githubusercontent.com/92897870/195410709-2865c90a-c558-427b-9f75-4ddc48631dd2.png)